app/views/partner_requests/new.html.erb

```diff
@@ -42,6 +42,7 @@
   <hr>
 
   <div class="actions">
+    <!-- TODO(chaserx): we should add some js to prevent submission if the items selected are the blank option or any item has an empty quantity -->
```

Comment:
Good thought on this, it was something I actually asked a couple of the diaper banks about and they'd like to receive requests (assuming they are items they carry) even if they are empty -- Often times they'll restock in-between getting an order and making a distribution.